portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
+            <div id="questions">
+
+                <!-- These fields will be populated by the about.js file -->
+
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first_img">
+                    </label>
+                    <p id="first_txt"></p>
+                </div>
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="second_answer" value="1"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="second_img">
+                    </label>
+                    <p id="second_txt"></p>
+                </div>
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="third_answer" value="2"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="third_img">
+                    </label>
+                    <p id="third_txt"></p>
+                </div>
+            </div>
+            <button id="two_truths" type="submit" onClick=next()>Start</button>
```

Comment:
I'd make this id more specific to what it is... something like "submit-guess-button" since it is a button that allows the user to submit their guess.

Er, now that I see that the button is used to start the game, I would recommend either of two options similar to my above comment for the area displaying either the start game prompt or the question number:
1)  Make the ID something that encapsulates both "start game" and "submit guess"; I would say something like "advance-game-button".  Or..
2)  Have two different buttons, and only turn the display on for one of them at a time, depending on whether the game has been started yet:  "start-game-button" and "submit-guess-button".

Reply:
Changed to advance-btn